### PR TITLE
added bulk upsert example

### PR DESCRIPTION
**Pull request by sureshdsk**
## What kind of change does this PR introduce?
Doc update
Added bulk upsert example to docs -  #3089

### CODE REVIEW

File: web/spec/supabase.yml [hunk 1]
```diff
@@ -675,6 +675,17 @@ pages:
             .from('messages')
             .upsert({ id: 3, message: 'foo', username: 'supabot' })
           ```
+      - name: Bulk Upsert your data
+        isSpotlight: true
```
Comment (kiwicopple) [inline web/spec/supabase.yml:679]: I think this one is better not as a spotlight (eg, make this `false`)

**Review by kiwicopple — CHANGES_REQUESTED** [review]
really nice addition to the docs @sureshdsk - just one change requested :)

File: web/spec/supabase.yml (hunk 1 above)
Comment (sureshdsk) [reply web/spec/supabase.yml]: @kiwicopple disabled spotlight.

Comment (kiwicopple) [reply web/spec/supabase.yml]: thanks!

**Review by kiwicopple — APPROVED** [review]

**Pull request merged**